[pulsar-client-tools] Ability to send a message with null body (#15145)
apache/pulsar: site2/docs/reference-cli-tools.md

|Flag|Description|Default|
|---|---|---|

> |`-f`, `--files`|Comma separated file paths to send, either -m or -f or --null must be specified.|[]|

Reviewer: Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null` .

Reviewer: ```suggestion
|`-f`, `--files`|Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null`.|[]|
```

Reviewer: ```suggestion
|`-f`, `--files`|Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null`.|[]|
```

> |`-f`, `--files`|Comma separated file paths to send, either -m or -f or --null must be specified.|[]|

Reviewer: Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null` .

> |`-m`, `--messages`|Messages to send, either -m or -f or --null must be specified. The default separator is comma|[]|

Reviewer: ```suggestion
|`-m`, `--messages`|Messages to send. At least one of the following values must be specified: `-m`, `-f` or `--null`. The default separator is a comma. |[]|
```
```suggestion
|`-m`, `--messages`|Messages to send, either -m or -f or --null must be specified. The default separator is comma|[]|
```

> |`-m`, `--messages`|Messages to send, either -m or -f or --null must be specified. The default separator is comma|[]|

Reviewer: ```suggestion
|`-m`, `--messages`|Messages to send. At least one of the following values must be specified: `-m`, `-f` or `--null`. The default separator is a comma. |[]|
```
```suggestion
|`-m`, `--messages`|Messages to send, either -m or -f or --null must be specified. The default separator is comma|[]|
```

|`-p`, `--properties`|Properties to add. If you want to add multiple properties, use the comma as the separator, e.g. `k1=v1,k2=v2`.| |
|`-ekn`, `--encryption-key-name`|The public key name to encrypt payload.| |
|`-ekv`, `--encryption-key-value`|The URI of public key to encrypt payload. For example, `file:///path/to/public.key` or `data:application/x-pem-file;base64,*****`.| |

> |`--null`|Send a message with the body is null|false|

Reviewer: ```suggestion
|`--null`|Send a message with a null body.|false|
```
